libs/product/src/facades/product/product-facade.interface.ts

```diff
@@ -0,0 +1,9 @@
+import { Observable } from 'rxjs';
+
+import { DaffProduct } from '../../models/product';
+
+export interface DaffProductFacadeInterface<T extends DaffProduct = DaffProduct> {
+	loading$: Observable<boolean>;
+	product$: Observable<T>;
+	dispatch: Function;
```

Comment:
Can this have a more explicit type than `Function`?

Reply:
done